app/code/Magento/InventorySales/Test/Integration/IsProductSalableForRequestedQty/IsAnySourceItemInStockConditionTest.php

```diff
@@ -69,6 +69,6 @@ public function testSourceItemsAreOutOfStock()
         $legacyStockItem->setBackorders(1);
         $legacyStockItem->setUseConfigBackorders(0);
         $this->stockItemRepository->save($legacyStockItem);
-        $this->assertFalse($this->isAnySourceInStockCondition->execute('SKU-1', 10, 1)->isSalable());
+        $this->assertFalse($this->isProductSalable->execute('SKU-1', 10, 1)->isSalable());
```

Comment:
Probably, we may use 'SKU-3' from Magento/InventoryApi/Test/_files/source_items.php and avoid creation of additional fixtures.